tests/FileTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Tests;
+
+use Rebilly\ApiKeyProvider;
+use Rebilly\Client;
+use Rebilly\Rest\File;
+use RuntimeException;
+
+/**
+ * Class FileTest.
+ *
+ * @group wip
+ */
+class FileTest extends TestCase
+{
+    public function testFile()
+    {
+        $client = new Client(['apiKey' => 'QWERTY']);
+
+        $client = new Client(
+            [
+                'apiKey' => ApiKeyProvider::env(),
+                'httpHandler' => function () use ($client) {
+                    $response = $client
+                        ->createResponse()
+                        ->withHeader('Content-Type', 'application/pdf')
+                    ;
```

Comment:
The semicolon on a separate line is not clearly defined by PSR-2 so developers are divided on this opinion... mostly they put it on a separate line just because of the possible diff as mentioned above regarding the trailing comma.

Because of that PHP code analizers has this as an option so it can be enabled or disabled... for example https://github.com/FriendsOfPHP/PHP-CS-Fixer has the options
> spaces_before_semicolon [symfony] - Single-line whitespace before closing semicolon are prohibited.
> multiline_spaces_before_semicolon [contrib] - Multi-line whitespace before closing semicolon are prohibited.

I'm not sure what is better and this sure is as other similar things a personal preference (like spaces or tabs, etc.) so I guess this should be a rebilly dev team decision so that all developers use the same style.